app/views/matches/show.html.erb

```diff
@@ -57,7 +57,7 @@
   <div class= "<%=team%>" >    
     
     <div class="team-score">
-      <%= link_to squad.name, squad %> - <%=players.map{ |pl| pl.gameweek_points[@match.gameweek] }.sum + captain.gameweek_points[@match.gameweek]%>
+      <%= link_to squad.name, squad %> - <%=players.map{ |pl| pl.gameweek_points[@match.gameweek]||0}.sum + (captain.gameweek_points[@match.gameweek]||0)%>
```

Comment:
this is right, but we want to make it more readable by putting spaces within curly brackets and either side of operators (this includes `||`)

making sure code is easily readable is just as important and writing code that works

once you've done this, we're ready to merge!